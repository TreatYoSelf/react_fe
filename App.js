import React from 'react';
import { StyleSheet, Text, View } from 'react-native';
import Landing from './src/components/Landing/Landing';

export default function App() {
  return (
    <View style={styles.container}>
      <Landing />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#fff',
    alignItems: 'center',
    justifyContent: 'center',
  },
});
