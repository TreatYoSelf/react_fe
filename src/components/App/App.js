import React from 'react';
import { StyleSheet, View } from 'react-native';
// import Landing from './src/components/Landing/Landing';
// import Profile from './src/components/Profile/Profile';
import Login from '../../containers/Login/Login';
import { Route } from "react-router-native";
import { NativeRouter } from "react-router-native";
import Calendar from '../Calendar/Calendar';
import Profile from '../Profile/Profile';
import PreferenceForm from '../../containers/PreferenceForm/PreferenceForm';

export default function App() {  
  return (
    <NativeRouter>
      <View style={styles.container}>
        <Route exact path="/" component={Login} />
        <Route exact path="/profile" component={Profile} /> 
        <Route exact path="/calendar" component={Calendar} /> 
      </View>
    </NativeRouter>
  );
}

// AppRegistry.registerComponent('MyApplication', () => App);

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: '#b6bfac',
    // alignItems: 'center',
    justifyContent: 'center',
    flexDirection: "column",
    height: "100%",
    },
});
