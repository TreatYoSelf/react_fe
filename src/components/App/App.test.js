import React from 'react';
import renderer from 'react-test-renderer';
import { MockedProvider } from '@apollo/react-testing';

import App from './App';

describe('<App />', () => {
    let tree;

    beforeEach(() => {
        tree = renderer.create(
            <MockedProvider addTypename={false}>
                <App />
            </MockedProvider>
        ).toJSON();
    })

    it('has 1 child', () => {
        expect(tree.children.length).toBe(1);
    });

    it('renders correctly', () => {
        expect(tree).toMatchSnapshot();
    });
});