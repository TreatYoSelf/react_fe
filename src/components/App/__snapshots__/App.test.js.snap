// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#fff",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#fff",
        "flex": 1,
        "justifyContent": "space-around",
        "margin": 50,
        "width": 300,
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#fff",
          "flex": 1,
          "justifyContent": "space-around",
          "margin": 50,
          "width": 300,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "height": 200,
            "width": 200,
          }
        }
      />
      <Text
        style={
          Object {
            "fontSize": 25,
          }
        }
      >
        Treat Yo Self
      </Text>
      <Text>
        An app where you can treat yourself to some personal time, guilt free.
      </Text>
      <View
        accessibilityRole="button"
        accessibilityStates={Array []}
        accessible={true}
        focusable={true}
        isTVSelectable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <View
          style={
            Array [
              Object {},
            ]
          }
        >
          <Text
            style={
              Array [
                Object {
                  "color": "#007AFF",
                  "fontSize": 18,
                  "padding": 8,
                  "textAlign": "center",
                },
              ]
            }
          >
            Sign in with Google
          </Text>
        </View>
      </View>
    </View>
  </View>
</View>
`;
